modules/oadp-using-legacy-aws-plugin.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-legacy-aws-plugin.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-using-legacy-aws-plugin_{context}"]
+= Using the legacy-aws Velero plugin in the DataProtectionApplication CR
+
+In the following use case, you configure the `DataProtectionApplication` CR with the `legacy-aws` Velero plugin and then back up an application.
+
+[NOTE]
+====
+Depending on the backup storage location you choose, you can use either the `legacy-aws` or the `aws` plugin in your `DataProtectionApplication` CR. Using both the plugins in the `DataProtectionApplication` CR will cause an error with the following message: `aws and legacy-aws can not be both specified in DPA spec.configuration.velero.defaultPlugins`.
+====
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have configured an {aws-short} S3-compatible object storage as a backup location.
+* You have an application with a database running in a separate namespace.
+
+.Procedure
+
+. Configure the `DataProtectionApplication` CR to use the `legacy-aws` Velero plugin as shown in the following example:
++
+.Example `DataProtectionApplication` CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - legacy-aws # <1>
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true 
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc 
+          s3ForcePathStyle: "true"
+          insecureSkipTLSVerify: "true"
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: oadp
+----
+<1> Use the `legacy-aws` plugin.
+<2> Specify the bucket name.
+
+. Create the `DataProtectionApplication` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <dpa_filename> 
+----
+
+. Verify that the `DataProtectionApplication` CR is created successfully by running the following command. In the example output, you can see the `status` object has the `type` field set to `Reconciled` and the `status` field set to `"True"`. This means, the `DataProtectionApplication` CR is successfully created.
```

Comment:
```suggestion
. Verify that the `DataProtectionApplication` CR is created successfully by running the following command. In the example output, you can see the `status` object has the `type` field set to `Reconciled` and the `status` field set to `"True"`. That status indicates that the `DataProtectionApplication` CR is successfully created.
```
IBM Style suggests rewriting sentences that start with "This" for clarity purposes: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references